howso/direct/client.py

```diff
@@ -2158,23 +2160,39 @@ def react_series(  # noqa: C901
         initial_values = serialize_cases(initial_values, initial_features,
                                          feature_attributes)
 
+        # All of these params must be of length 1 or N
+        # where N is the length of the largest
+        one_or_more_params = [
+            contexts,
+            initial_values,
+            serialized_continue_series_values,
+            serialized_series_context_values,
+            case_indices,
+            actions,
+            max_series_lengths,
+            series_stop_maps,
+        ]
+        if any(one_or_more_params):
+            param_lengths = set([len(x) for x in one_or_more_params if x])
+            if len(param_lengths - {1}) > 1:
```

Comment:
How about this:

`if max([len(x) for x in one_or_more_params if x]) > 1:`

Reply:
There can be lengths greater than 1, but there can only be one value of lengths greater than one.

So if more than one of these parameters is of length > 1, then they must be the same length.
My code is attempting to raise a ValueError if there are 2 or more lengths greater than one but unequal